src/test/java/com/redhat/insights/agent/InsightsAgentHttpClientTest.java

```diff
@@ -31,4 +43,57 @@ void testBearerToken() {
             .anyMatch(
                 h -> h.getName().equals("Authorization") && h.getValue().equals("Bearer bar")));
   }
+
+  @Test
+  void testBearerToken2() {
+    Optional<AgentConfiguration> oConfig = parseArgs("name=foo;token=bar;is_ocp=true");
+    InsightsAgentHttpClient client = new InsightsAgentHttpClient(oConfig.get());
+    HttpPost post = client.createAuthTokenPost();
+    List<Header> headers = Arrays.asList(post.getAllHeaders());
+    assertTrue(
+        headers.stream()
+            .anyMatch(
+                h -> h.getName().equals("Authorization") && h.getValue().equals("Bearer bar")));
+  }
+
+  @Test
+  void sendInsightsReportOk(WireMockRuntimeInfo wmri) {
+    stubFor(post(InsightsConfiguration.DEFAULT_UPLOAD_URI).willReturn(aResponse().withStatus(201)));
+    Optional<AgentConfiguration> oConfig =
+        parseArgs("name=foo;token=fubar;base_url=" + wmri.getHttpBaseUrl());
+    final InsightsReport report = AgentBasicReport.of(oConfig.get());
+    report.generateReport(Filtering.DEFAULT);
+    InsightsAgentHttpClient client = new InsightsAgentHttpClient(oConfig.get());
+    client.sendInsightsReport("foo", report);
+  }
```

Comment:
This test doesn't seem to have any assertions.

Reply:
Now verifying that a request was indeed made.